staging/src/k8s.io/component-base/metrics/testutil/metrics.go

```diff
@@ -360,3 +360,13 @@ func GetHistogramMetricValue(m metrics.ObserverMetric) (float64, error) {
 	}
 	return metricProto.Histogram.GetSampleSum(), nil
 }
+
+// LabelsMatch returns true if metric has all expected labels otherwise false
+func LabelsMatch(metric *dto.Metric, labelFilter map[string]string) bool {
+	for _, lp := range metric.GetLabel() {
+		if value, ok := labelFilter[lp.GetName()]; ok && lp.GetValue() != value {
+			return false
+		}
+	}
+	return true
+}
```

Comment:
The docstring looks like it's not true. Let's say we have a labelFilter that this:

```
map[string]string{
    "a": "expectedVal1",
    "b": "expectedVal2",
}
```
If the metric only has a single label, let's say 'a', then this function returns true, which seems not in line with the documentation.

Reply:
oo , sure .
Thanks . I will update it .

Reply:
/cc @logicalhan 
/cc @RainbowMango 
could you review it again?Thanks